src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,57 +70,176 @@ contract DssSpellCollateralAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
 
-    function onboardNewCollaterals() internal {
-        // ----------------------------- Collateral onboarding -----------------------------
-        //  Add ______________ as a new Vault Type
-        //  Poll Link:
-
-        // DssExecLib.addNewCollateral(
-        //     CollateralOpts({
-        //         ilk:                   ,
-        //         gem:                   ,
-        //         join:                  ,
-        //         clip:                  ,
-        //         calc:                  ,
-        //         pip:                   ,
-        //         isLiquidatable:        ,
-        //         isOSM:                 ,
-        //         whitelistOSM:          ,
-        //         ilkDebtCeiling:        ,
-        //         minVaultAmount:        ,
-        //         maxLiquidationAmount:  ,
-        //         liquidationPenalty:    ,
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    uint256 internal constant ZERO_PCT_RATE                  = 1000000000000000000000000000;
+
+    // --- Math ---
+    uint256 internal constant WAD = 10**18;
+
+    // -- RWA007 MIP21 components --
+    address internal constant RWA007                         = 0xD063270642ff718DA1c58E12BD6a2598f7e874B3;
+    address internal constant MCD_JOIN_RWA007_A              = 0x9C9E33E22b683F789411288497f8DC560f1F0466;
+    address internal constant RWA007_A_URN                   = 0xa1b1D392fCB99F8B39c7530a599bCfcd2f1fB22f;
```

Comment:
I think this deployer here is missing in `addresses_deployers.sol` and maybe others from previous rwa onboarding.

Reply:
@amusingaxl Can you please confirm this

Reply:
done